Cannot build Core

Hi!

The following error happens when trying to build (openSUSE Tumbleweed, using openJDK 8):
```
[INFO] Reactor Summary for JDT Language Server :: Parent 0.35.0-SNAPSHOT:
[INFO] 
[INFO] JDT Language Server :: Parent ...................... SUCCESS [  0.135 s]
[INFO] JDT Language Server :: Target Platform ............. SUCCESS [  0.203 s]
[INFO] JDT Language Server :: Core ........................ FAILURE [  6.352 s]
[INFO] JDT Language Server :: Tests ....................... SKIPPED
[INFO] JDT Language Server :: Product ..................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:03 min
[INFO] Finished at: 2019-03-09T22:51:31+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:1.3.0:compile (default-compile) on project org.eclipse.jdt.ls.core: Compilation failure: Compilation failure: 
[ERROR] /home/emilio/Prog/Java/eclipse.jdt.ls/org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/PreferenceManager.java:[221] 
[ERROR] 	pref.put(JavaBuilder.PREF_NULL_SCHEDULING_RULE, stringValue);
[ERROR] 	                     ^^^^^^^^^^^^^^^^^^^^^^^^^
[ERROR] PREF_NULL_SCHEDULING_RULE cannot be resolved or is not a field
[ERROR] 1 problem (1 error)
```

As I have seen the above mentioned field has been removed from JavaBuilder.java in this commit:
https://github.com/eclipse/eclipse.jdt.core/commit/f130788d4def96e08510f33bfc111f61e9a63fd9

Cheers!

--- a/org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/PreferenceManager.java
+++ b/org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/PreferenceManager.java
@@ -35,7 +35,6 @@ import org.eclipse.jdt.core.JavaCore;
 import org.eclipse.jdt.core.formatter.DefaultCodeFormatterConstants;
 import org.eclipse.jdt.core.manipulation.CodeStyleConfiguration;
 import org.eclipse.jdt.core.manipulation.JavaManipulation;
-import org.eclipse.jdt.internal.core.builder.JavaBuilder;
 import org.eclipse.jdt.internal.core.manipulation.CodeTemplateContextType;
 import org.eclipse.jdt.internal.core.manipulation.MembersOrderPreferenceCacheCommon;
 import org.eclipse.jdt.internal.core.manipulation.StubUtility;
@@ -218,7 +217,6 @@ public class PreferenceManager {
 		IEclipsePreferences pref = InstanceScope.INSTANCE.getNode(IMavenConstants.PLUGIN_ID);
 		pref.put(MavenPreferenceConstants.P_BUILDER_USE_NULL_SCHEDULING_RULE, stringValue);
 		pref = InstanceScope.INSTANCE.getNode(JavaCore.PLUGIN_ID);
-		pref.put(JavaBuilder.PREF_NULL_SCHEDULING_RULE, stringValue);
 	}
 
 	/**

